src/Account/StudentTopicPreferences.tsx

```diff
@@ -0,0 +1,103 @@
+import { Button, FormControlLabel, Grid, Switch } from '@material-ui/core';
+import React from 'react';
+import { Controller, useForm } from 'react-hook-form';
+import _ from 'lodash';
+import { nameof } from '../Utilities/TypescriptUtils';
+import localPreferences from '../Utilities/LocalPreferences';
+import useAlertState from '../Hooks/useAlertState';
+import { Alert } from 'react-bootstrap';
+
+const { topicPreferences } = localPreferences;
+
+export interface StudentTopicPreferencesInputs {
+    useSeconds: boolean;
+}
+
+interface StudentTopicPreferencesProps {
+
+}
+
+
+export const StudentTopicPreferences: React.FC<StudentTopicPreferencesProps> = () => {
+    const [alert, setAlert] = useAlertState();
+    const preferences = useForm<StudentTopicPreferencesInputs>({
+        mode: 'onSubmit',
+        shouldFocusError: true,
+        defaultValues: {
+            useSeconds: topicPreferences.useSeconds
+        }
+    });
+    const { control, getValues, handleSubmit, watch } = preferences;
+    const useSeconds = watch('useSeconds');
+
+    const onSubmit = (values: StudentTopicPreferencesInputs) => {
+        topicPreferences.useSeconds = values.useSeconds;
+        setAlert({
+            variant: 'success',
+            message: 'Saved Successful'
+        });
+    };
+
+    const clearAlert = () => {
+        setAlert({
+            variant: 'info',
+            message: ''
+        });
+    };
+
+    return (<Grid container item spacing={3} xs={6} justify='center'>
+        <form onChange={() => clearAlert()} onSubmit={handleSubmit(onSubmit)}>
+            <Alert variant={alert.variant} show={alert.message.length > 0}>{alert.message}</Alert>
+            <Grid item md={12}>
+                <h1>Topic Preferences</h1>
+                <p>These preferences are stored locally. If you clear browser storage, switch browsers or use another computer they are subject to change.</p>
+            </Grid>
+            <Grid item md={12}>
+                <p>
+                    <strong>Use Seconds Countdown</strong> when on will display <em>hours:minutes:seconds</em> on the day the topic is due (example <em>Due in 5:09:04</em>). 
```

Comment:
```suggestion
                    <strong>Use Seconds Countdown</strong>, when on, will display <em>hours:minutes:seconds</em> on the day the topic is due (example <em>Due in 5:09:04</em>). 
```

Reply:
It would also be cool to just pull the component in so they could see what it would look like with a target that was always an hour in the future or something.